subprojects/docs/src/docs/userguide/migration/upgrading_version_8.adoc

```diff
@@ -44,6 +44,22 @@ The previous step will help you identify potential problems by issuing deprecati
 
 === Potential breaking changes
 
+==== Changes to paths of included builds
+
+In order to handle conflicts between nested included build names better, Gradle now uses the directory hierarchy of included builds to assign the build path.
+If you are running tasks from the command line in nested included builds, then you may need to adjust your invocation.
+
+For example, if you have the following hierarchy:
```

Comment:
I think you should also specific which of these are project dirs and perhaps also show the `settings.gradle` file.

Reply:
How about now?

Reply:
I think this is much clearer. 👍